### PR TITLE
B-19887: Add pricing estimate column to MTO Service Items

### PR DESCRIPTION
**Pull request by TevinAdams**
## Summary

In this story the pricing_estimate column is being added to mto_service_items table.

### How to test

1. Run the migration scripts
2. Check the mto_service_items table for the pricing_estimate column

### CODE REVIEW

File: migrations/app/schema/20240516143952_add_pricing_estimate_to_mto_service_items.up.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE mto_service_items ADD COLUMN pricing_estimate integer DEFAULT NULL;
```
Comment (danieljordan-caci) [inline migrations/app/schema/20240516143952_add_pricing_estimate_to_mto_service_items.up.sql:1]: Add `IF NOT EXISTS` and comments please

**Review by danieljordan-caci — CHANGES_REQUESTED** [review]

**Review by traskowskycaci — CHANGES_REQUESTED** [review]
New column pops up for me locally! 

First time I've done this as a requested change but since I thought of it - can you update the DB tracking spreadsheet? Then I'll get you the green stamp!

File: migrations/app/schema/20240516143952_add_pricing_estimate_to_mto_service_items.up.sql (hunk 1 above)
Comment (TevinAdams) [reply migrations/app/schema/20240516143952_add_pricing_estimate_to_mto_service_items.up.sql]: @danieljordan-caci Done

**Comment by TevinAdams** [comment]
> New column pops up for me locally!
> 
> First time I've done this as a requested change but since I thought of it - can you update the DB tracking spreadsheet? Then I'll get you the green stamp!
@traskowskycaci  Done

**Review by danieljordan-caci — APPROVED** [review]

**Review by traskowskycaci — APPROVED** [review]
Excellent - thank you!

**Review by cameroncaci — APPROVED** [review]

**Pull request merged**